commands/whoami.go

```diff
@@ -3,13 +3,49 @@ package commands
 import (
 	"fmt"
 	tgbotapi "github.com/go-telegram-bot-api/telegram-bot-api/v5"
+	"github.com/sxyazi/bendan/utils"
+	"io"
+	"log"
+	"net/http"
+	"regexp"
 )
 
+// This DC query method is based on checking the user profile photos.
```

Comment:
I made a grammar mistake when writing comments, the photos here should be photo (13-14 line).